### PR TITLE
General overhaul of the HTML standard

### PR DESCRIPTION
**Pull request by pzi**
Yep... that happened... :pencil:

Applied some of that polish:

### CODE REVIEW

File: html.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
+# HTML / Haml
 
-We use HAML at The Frontier Group, ERB is only acceptable for mailers, plain text and non-HTML rendering.
+We use Haml at The Frontier Group, ERB is only acceptable for mailers, plain text and non-HTML rendering.
+
+## Table of Contents
+
+* [Syntax](#syntax)
+* [Haml Initializer](#haml-initializer)
+    * [Adding the initializer to your project](#adding-the-initializer-to-your-project)
```
Comment (jgunnink) [inline html.md]: Why so indent? Compared to other indenting on lines 14 and 15 for example 😄

**Comment by colynprater** [comment]
Awesome 👍

**Comment by BenR1312** [comment]
K 👍

File: html.md [hunk 2]
```diff
@@ -58,28 +71,45 @@ Make sure to specify the encoding in the HTML templates and documents right befo
...
+Loading fallback scripts from a CDN for older IE versions could be beneficial as it may have been cached by the browser before.
 
 ## Semantics
 
 ### Use HTML according to its purpose.
 
-Use elements for what they have been created for. For example, use heading elements for headings, `p` elements for paragraphs, `a` elements for anchors, etc.
+Use elements for what they have been created for. For example, use heading elements for headings, `p` elements for paragraphs, `a` elements for anchors, etc. See [HTML element reference on MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element) for a detailed list of available elements.
+
+An exemption is the `<i>` element. We typically use `i` to display icons. This way we have full control over a pseudo "icon element" and can attach multiple classes and other HTML attributes to it if necessary.
```
Comment (psyren89) [inline html.md]: `exception`

File: html.md [hunk 3]
```diff
@@ -58,28 +71,45 @@ Make sure to specify the encoding in the HTML templates and documents right befo
...
+**Note:** The same applies to [Respond.js](https://github.com/scottjehl/Respond) if required.
+
+Loading fallback scripts from a CDN for older IE versions could be beneficial as it may have been cached by the browser before.
 
 ## Semantics
 
 ### Use HTML according to its purpose.
 
-Use elements for what they have been created for. For example, use heading elements for headings, `p` elements for paragraphs, `a` elements for anchors, etc.
+Use elements for what they have been created for. For example, use heading elements for headings, `p` elements for paragraphs, `a` elements for anchors, etc. See [HTML element reference on MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element) for a detailed list of available elements.
```
Comment (psyren89) [inline html.md]: `Use elements for their intended purpose`

Comment (sheldongill) [inline html.md]: intended purpose _only_.

File: html.md (hunk 2 above)
Comment (sheldongill) [inline html.md]: The one exception

File: html.md [hunk 4]
```diff
@@ -58,28 +71,45 @@ Make sure to specify the encoding in the HTML templates and documents right befo
...
+An exemption is the `<i>` element. We typically use `i` to display icons. This way we have full control over a pseudo "icon element" and can attach multiple classes and other HTML attributes to it if necessary.
+
+Example: `%i.ico.ico-large.ico-user{aria: {hidden: "true"}}`.
+
+**Note:** With the addition of the `aria-hidden` attribute, we hide this element and its content from assistive technology (ie. screenreaders).
 
 ### Provide alternative contents for images
 
-Providing alternative contents is important for accessibility reasons: a blind user has few cues to tell what an image is about without `alt` attribute.
+Providing alternative contents is important for accessibility reasons: a visually impaired user has limited cues to tell what an image is about without `alt` attribute.
```
Comment (sheldongill) [inline html.md]: Providing alternative content is important

**Pull request merged**